run_jobs.py

```diff
@@ -125,7 +123,12 @@ def run_jobs(self) -> None:
                     logger.error('MySQL was not available')
                     sys.exit(1)
                 else:
-                    logger.debug('Still waiting!')
+                    if i == 1:
```

Comment:
nit: If the loop range is changed to start with 0 (as suggested in my other comment), then this condition should check for that…

```suggestion
                    if i < 1:
```

or…

```suggestion
                    if not i:
```

(I wanted to suggest changing the range initial value to 0 when I wrote the issue, but I thought it might be seen as nitpicky.  But in this case, using 0 actually lets us keep the original loop count.)

Reply:
I chose to use `if i == 0`, not to be contrarian, I just think it's a bit more specific? I dunno, it's probably fine any of these ways.

Reply:
I like the `not i` solution, because it seems cool.  There's a slight feeling of risk associated with it, though, isn't there?  The `== 0` solution is fine and risk free, but `< 1` is a character shorter.  (Whee!)